### ISSUE TITLE
[Ubuntu 1.6.4] Xorg crashes when scrolling down the boards manager

### ISSUE THREAD

**Issue by NicoHood** [arduino/Arduino#3174]
I have several boards installed within my sketchbook/hardware folder. Tried with my HoodLoader2, HID-Project and the sparkfun folder.

Thats a lot of boards, so you have to scroll down to reach the last entry. However if you try to scroll down the whole X session crashes and you have to login again. Every program like browser crashes as well.

Tried with lastest standard Ubuntu 64 bit, IDE 1.6.4 and java7. I did not test any older release yet.

**Comment by ffissore** [comment]
Does it happen with latest nightly?

**Comment by NicoHood** [comment]
Yes, problem still there. X_x

**Comment by ffissore** [comment]
Confirmed. Veeery annoying.

**Comment by ffissore** [comment]
Fixed. Fix will be available with next hourly build

**Comment by ffissore** [comment]
Nope, bug is still there

**Comment by ffissore** [comment]
Various attempts to solve the issue failed. However, when running the IDE with latest java 8, the issues did not show up. I then suggest you to install latest java 8 following [this tutorial for ubuntu](http://ubuntuhandbook.org/index.php/2015/01/install-openjdk-8-ubuntu-14-04-12-04-lts/) or downloading it directly from [oracle website](http://www.oracle.com/technetwork/java/javase/downloads/index.html) and changing the `java` command [here](https://github.com/arduino/Arduino/blob/master/build/linux/dist/arduino#L31) specifying the full path (mine is `/opt/jvm/jdk1.8.0/bin/java`)

**Comment by NicoHood** [comment]
The problem is still there.
Telling the people to install java8 and pretend the bug is fixed is not a solution. I suggest two things:

Add a note to the installing instructions/download section that you need java 8 (not tested for me though)
Or open the issue and give away some cash if someone else is able to fix it.

Its a critical bug, all programs crash, all data is gone because of this fucking IDE. If someone accidentally gets this bug, this will drive him crazy. Especially when you havent saved your code/docs.

I noticed some changes in the nightly (of course). It would be nice if you could pin the Arduino IDE to the Ubuntu launcher as well. I've created a shortcut myself but it will open a 2nd Icon on the launcher.

**Comment by ffissore** [comment]
Please avoid using harsh words. If the problem lies in how the jvm handles graphics, we can't do much but ask to use an updated jvm.

**Comment by NicoHood** [comment]
It wasnt ment to be harsh. I just remembered how I was surprised when my X Server closed the first time.

Do I have to uninstall java 7 first?
I will try that and let you know if it works. You should add those notes somewhere then.

**Comment by ffissore** [comment]
Linux version will bundle the JVM, thus solving this issue

**Comment by NicoHood** [comment]
I had this program today again.
I had 1.6.6 open and also started 1.6.5 at the same time. Both via commandline, but sadly didnt log the output.

1.6.5 opened with a few windows, I closed them and then the whole screen died. Music went off after a while and I had to relogin with a new clean session. Same as the issue above.
As far as I can see java8 is already bundled in 1.6.5? I have at least not manually installed java.
Using Elementary os now, based on ubuntu 14.04

Here is my xsession-errors.old. Thats all I have right now. This could be a new problem, but related to this issue. I had it only once so I did not open a new issue for now.

I think the crash starts after the Rythmbox Robin Schulz logs:
```
Script for ibus started at run_im.
Script for auto started at run_im.
Script for default started at run_im.
Script for ibus started at run_im.
Script for auto started at run_im.
Script for default started at run_im.

** (gnome-session:1688): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused

** (gnome-session-check-accelerated:1820): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused
gnome-session[1688]: WARNING: Error while executing session-migration: Failed to execute child process "session-migration" (No such file or directory)
gnome-session[1688]: WARNING: Could not parse desktop file user-dirs-update-gtk.desktop or it references a not found TryExec binary
gnome-session[1688]: WARNING: Could not parse desktop file user-dirs-update-gtk-pantheon.desktop or it references a not found TryExec binary

** (gnome-settings-daemon:1828): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): keyboard-plugin-WARNING **: Unhandled settings change, key 'input-sources-switcher'

(gnome-settings-daemon:1828): keyboard-plugin-WARNING **: Unhandled settings change, key 'bell-custom-file'

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): keyboard-plugin-WARNING **: Unhandled settings change, key 'input-sources-switcher'

(gnome-settings-daemon:1828): keyboard-plugin-WARNING **: Unhandled settings change, key 'bell-custom-file'

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): keyboard-plugin-WARNING **: Unhandled settings change, key 'input-sources-switcher'

(gnome-settings-daemon:1828): keyboard-plugin-WARNING **: Unhandled settings change, key 'bell-custom-file'

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): media-keys-plugin-WARNING **: Grab failed for some keys, another application may already have access the them.

(gnome-settings-daemon:1828): color-plugin-WARNING **: failed to get edid: unable to get EDID for output

(gnome-settings-daemon:1828): color-plugin-WARNING **: unable to get EDID for xrandr-eDP1: unable to get EDID for output
Window manager warning: Log level 128: Settings.vala:158: Loading settings from schema 'org.gnome.desktop.background'
Window manager warning: Log level 128: Settings.vala:158: Loading settings from schema 'org.pantheon.desktop.gala.appearance'
Window manager warning: Log level 128: Settings.vala:158: Loading settings from schema 'org.pantheon.desktop.gala.keybindings'
Window manager warning: Log level 128: Settings.vala:158: Loading settings from schema 'org.pantheon.desktop.gala.shadows'
Window manager warning: Log level 128: Settings.vala:158: Loading settings from schema 'org.pantheon.desktop.gala.behavior'
Window manager warning: Log level 128: PluginManager.vala:132: Loaded plugin Mask Corners (libgala-maskcorners.so)
Window manager warning: Log level 128: PluginManager.vala:132: Loaded plugin Notify (libgala-notify.so)
Window manager warning: Log level 128: PluginManager.vala:132: Loaded plugin Zoom (libgala-zoom.so)
Window manager warning: Log level 128: Settings.vala:158: Loading settings from schema 'org.pantheon.desktop.gala.mask-corners'
Window manager warning: Log level 128: Preferences.vala:340: Loading preferences from file '/home/linuxuser/.config/plank/dock1/settings'
Window manager warning: Log level 16: Preferences.vala:186: '/usr/share/themes/elementary/plank/dock.theme' is read-only!
Window manager warning: Log level 128: Preferences.vala:340: Loading preferences from file '/usr/share/themes/elementary/plank/dock.theme'

** (nm-applet:1980): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused

** (light-locker:1989): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

** (polkit-gnome-authentication-agent-1:1999): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused
initctl: UPSTART_SESSION isn't set in the environment. Unable to locate the Upstart instance.

(process:1986): libnotify-WARNING **: Failed to connect to proxy

** (process:1986): CRITICAL **: volume_control_set_volume_internal: assertion '_tmp1_ == PA_CONTEXT_READY' failed

** (process:1986): CRITICAL **: file /build/buildd/indicator-sound-12.10.2+14.04.20140401/obj-x86_64-linux-gnu/src/volume-control.c: line 1787: uncaught error: GDBus.Error:org.freedesktop.DBus.Error.InvalidArgs: No such interface (g-dbus-error-quark, 16)

** (plank:2030): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused

** (slingshot-launcher:2034): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x1a00003 (wingpanel)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Log level 128: Settings.vala:158: Loading settings from schema 'org.pantheon.desktop.gala.animations'

(Pidgin:2214): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:2214): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:2214): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:2214): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:2214): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"

** (gnome-user-share:2316): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused

** (zeitgeist-datahub:2331): WARNING **: zeitgeist-datahub.vala:229: Unable to get name "org.gnome.zeitgeist.datahub" on the bus!

(Pidgin:2376): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:2376): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:2376): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:2376): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:2376): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"
Exiting because another libpurple client is already running.

(process:2435): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:2435): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:2435): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:2435): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:2435): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:2435): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"

** (update-notifier:2514): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused

(process:2530): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(thunderbird:2530): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(thunderbird:2530): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(thunderbird:2530): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(thunderbird:2530): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(thunderbird:2530): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x220004b (Buddy List)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (#arduino)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x26000a8 (ITO (Indiu)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

** (rhythmbox:2623): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused

(rhythmbox:2623): Gtk-CRITICAL **: gtk_css_provider_load_from_path: assertion 'path != NULL' failed

(rhythmbox:2623): Gtk-WARNING **: Duplicate child name in GtkStack: Add to Playlist


(rhythmbox:2623): Gtk-WARNING **: Duplicate child name in GtkStack: Add to Playlist


(rhythmbox:2623): Gtk-WARNING **: Duplicate child name in GtkStack: Add to Playlist


(rhythmbox:2623): Gtk-WARNING **: Duplicate child name in GtkStack: Add to Playlist


(rhythmbox:2623): Gtk-WARNING **: Duplicate child name in GtkStack: Add to Playlist


(rhythmbox:2623): Gtk-WARNING **: Duplicate child name in GtkStack: Add to Playlist

Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2800007 (Rhythmbox)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2800007 (Rhythmbox)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Log level 128: Settings.vala:158: Loading settings from schema 'org.pantheon.desktop.gala.notifications'

(gnome-settings-daemon:1828): color-plugin-WARNING **: unable to get EDID for xrandr-eDP1: unable to get EDID for output
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x1c00003 (Slingshot)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

** (pavucontrol:2708): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-8Pse0Q6lVX: Connection refused
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2800007 (K.I.Z. - A)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2e0000a ()
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2800007 (K.I.Z. - A)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2800007 (Depeche Mo)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2c00003 (Volume Con)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(gnome-settings-daemon:1828): color-plugin-WARNING **: failed to connect to device: Failed to connect to missing device /org/freedesktop/ColorManager/devices/cups_Canon_iP4800

(gnome-settings-daemon:1828): color-plugin-WARNING **: failed to connect to device: Failed to connect to missing device /org/freedesktop/ColorManager/devices/cups_iP4800_series

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2800007 (Eminem - W)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2800007 (Fettes Bro)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2800007 (Black Eyed)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2800007 (Black Eyed)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x26000a8 (Copper Foi)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2a003ec (Mail Serve)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2a003ec (Mail Serve)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x220004b (Buddy List)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (#arduino)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x26000a8 (De Morgan')
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x320031e (/dev/ttyAC)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x320008f (Fibonacci2)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x26000a8 (ITO covere)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2a003ec (Mail Serve)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2a003ec (Mail Serve)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(process:7517): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:7517): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7517): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7517): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7517): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7517): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"

(process:7720): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:7720): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7720): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7720): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7720): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7720): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"

(process:7773): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:7773): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7773): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7773): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7773): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:7773): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"

(process:8187): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:8187): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8187): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8187): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8187): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8187): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"

(process:8297): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:8297): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8297): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8297): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8297): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8297): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"

(process:8463): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:8463): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8463): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8463): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8463): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8463): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x320031e (/dev/ttyAC)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x320008f (Fibonacci2)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(Pidgin:9433): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:9433): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:9433): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:9433): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:9433): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"
Exiting because another libpurple client is already running.

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(gnome-settings-daemon:1828): color-plugin-WARNING **: unable to get EDID for xrandr-eDP1: unable to get EDID for output

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(process:13805): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:13805): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:13805): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:13805): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:13805): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:13805): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"

(process:14053): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:14053): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:14053): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:14053): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:14053): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:14053): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"

(process:15227): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:15227): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:15227): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:15227): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:15227): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:15227): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x26000a8 (C# coding )
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(firefox:2435): Gtk-CRITICAL **: IA__gtk_clipboard_set_with_data: assertion 'targets != NULL' failed

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Taking Me Home (Radio Mix)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Taking Me Home (Radio Mix)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Whatever (Radio Mix)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Whatever (Radio Mix)): Timeout was reached

(gnome-settings-daemon:1828): color-plugin-WARNING **: unable to get EDID for xrandr-eDP1: unable to get EDID for output
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (##c++)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug

*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug


(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x360000a (Visualizer)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x360000a (MSGEQ7.h ()
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (##c++)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2a000a8 (Inbox - ma)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(process:8295): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:8295): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8295): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8295): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8295): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8295): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2a000a8 (Inbox - ma)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(process:8364): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed

(firefox:8364): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8364): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8364): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8364): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(firefox:8364): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2a000a8 (Inbox - ma)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (##c++)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x26000a8 (Remix Mini)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2a000a8 (Inbox - ma)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x26000a8 (Remix Mini)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (##c++)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(Pidgin:8749): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:8749): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:8749): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:8749): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",

(Pidgin:8749): Gtk-WARNING **: Unable to locate theme engine in module_path: "pixmap",
Gtk-Message: Failed to load module "canberra-gtk-module"
Exiting because another libpurple client is already running.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x26000a8 (Remix Mini)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x220004b (Buddy List)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (##c++)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (In The Morning Light (Radio Mix)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (In The Morning Light (Radio Mix)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Whatever (Radio Mix)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Whatever (Radio Mix)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Summer Nights (Radio Mix)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Summer Nights (Radio Mix)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Changes (Robin Schulz Remix - Radio Edit)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Changes (Robin Schulz Remix - Radio Edit)): Timeout was reached

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (No Rest For The Wicked (Robin Schulz Edit)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (No Rest For The Wicked (Robin Schulz Edit)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Whatever (Radio Mix)): Timeout was reached

(rhythmbox:2623): Rhythmbox-WARNING **: Failed to send notification (Whatever (Radio Mix)): Timeout was reached

(gnome-settings-daemon:1828): color-plugin-WARNING **: unable to get EDID for xrandr-eDP1: unable to get EDID for output
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2a000a8 (Inbox - ma)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x220004b (Buddy List)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (##c++)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x320008f (Fibonacci2)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x320031e (/dev/ttyAC)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (##c++)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.

(process:1982): GLib-CRITICAL **: g_variant_get_string: assertion 'value != NULL' failed
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x26000a8 (Data struc)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (##c++)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x32085ce (/dev/ttyAC)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x320008f (Fibonacci2)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x26000a8 (Issues · )
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2a000a8 (Inbox - ma)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (#arduino)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x32085ce (/dev/ttyAC)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x320008f (Fibonacci2)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2200177 (#arduino)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2e0000a (./arduino)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: Buggy client sent a _NET_ACTIVE_WINDOW message with a timestamp of 0 for 0x2e0000a (./arduino)
Window manager warning: meta_window_activate called by a pager with a 0 timestamp; the pager needs to be fixed.
Window manager warning: 
(nm-applet:1980): Gdk-WARNING **: nm-applet: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.

Log level 16: gala: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.


(polkit-gnome-authentication-agent-1:1999): Gdk-WARNING **: polkit-gnome-authentication-agent-1: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.


(pavucontrol:2708): Gdk-WARNING **: pavucontrol: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.

gnome-session[1688]: Gdk-WARNING: gnome-session: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.

firefox: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.
Pidgin: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.
thunderbird: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.

(gnome-settings-daemon:1828): Gdk-WARNING **: gnome-settings-daemon: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.


(light-locker:1989): Gdk-WARNING **: light-locker: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.


(rhythmbox:2623): Gdk-WARNING **: rhythmbox: Fatal IO error 104 (Connection reset by peer) on X server :0.


(update-notifier:2514): Gdk-WARNING **: update-notifier: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.

** Message: ProcessWrapper.vala:99: Process 'wingpanel' watch exit
** Message: ProcessWrapper.vala:111: Process wingpanel is being terminated
** Message: ProcessWrapper.vala:123: ET = 18622.265903 secs	Min allowed time = 3.500000
** Message: ProcessWrapper.vala:46: Crash count of 'wingpanel' has been reset
** Message: ProcessWrapper.vala:99: Process 'plank' watch exit
** Message: ProcessWrapper.vala:111: Process plank is being terminated
** Message: ProcessWrapper.vala:123: ET = 18622.271308 secs	Min allowed time = 3.500000
** Message: ProcessWrapper.vala:46: Crash count of 'plank' has been reset
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Indicator-Datetime-Message: indicator-datetime exiting; failed/lost bus ownership
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.

** (process:1986): CRITICAL **: indicator_sound_service_name_lost: assertion 'connection != NULL' failed
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.

** (process:1979): CRITICAL **: service_on_name_lost: assertion 'connection != NULL' failed

** (process:2018): WARNING **: exiting: service couldn't acquire, or lost ownership of, busname
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.

(process:1992): indicator-application-service-CRITICAL **: Unable to send ApplicationRemoved signal: The connection is closed

(process:1992): indicator-application-service-WARNING **: Name Lost
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.

(deja-dup-monitor:2603): GVFS-RemoteVolumeMonitor-WARNING **: Owner of volume monitor org.gtk.Private.UDisks2VolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(deja-dup-monitor:2603): GVFS-RemoteVolumeMonitor-WARNING **: Owner of volume monitor org.gtk.Private.AfcVolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(deja-dup-monitor:2603): GVFS-RemoteVolumeMonitor-WARNING **: Owner of volume monitor org.gtk.Private.GPhoto2VolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(deja-dup-monitor:2603): GVFS-RemoteVolumeMonitor-WARNING **: Owner of volume monitor org.gtk.Private.MTPVolumeMonitor disconnected from the bus; removing drives/volumes/mounts
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.

(process:1986): GLib-GObject-CRITICAL **: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
Invalid MIT-MAGIC-COOKIE-1 keyInvalid MIT-MAGIC-COOKIE-1 key
** (plank:19196): WARNING **: Could not open X display
Invalid MIT-MAGIC-COOKIE-1 keyInvalid MIT-MAGIC-COOKIE-1 keyUnable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(plank:19196): Gtk-WARNING **: cannot open display: :0
```

**Comment by NicoHood** [comment]
/var/log/Xorg.o.log.old (last lines)
```
[ 17261.960] (II) AIGLX: Resuming AIGLX clients after VT switch
[ 17261.960] (II) intel(0): switch to mode 1600x900@60.0 on eDP1 using pipe 0, position (0, 0), rotation normal, reflection none
[ 17262.772] (II) intel(0): switch to mode 1280x1024@60.0 on VGA1 using pipe 1, position (1600, 0), rotation normal, reflection none
[ 17262.821] (--) synaptics: SynPS/2 Synaptics TouchPad: touchpad found
[ 17262.885] (II) XKB: reuse xkmfile /var/lib/xkb/server-81907AB388B478D711E6423DD8B0048239BE0CB6.xkm
[ 17262.907] (II) XKB: reuse xkmfile /var/lib/xkb/server-81907AB388B478D711E6423DD8B0048239BE0CB6.xkm
[ 17262.914] (II) XKB: reuse xkmfile /var/lib/xkb/server-81907AB388B478D711E6423DD8B0048239BE0CB6.xkm
[ 17262.921] (II) XKB: reuse xkmfile /var/lib/xkb/server-81907AB388B478D711E6423DD8B0048239BE0CB6.xkm
[ 17262.931] (II) XKB: reuse xkmfile /var/lib/xkb/server-81907AB388B478D711E6423DD8B0048239BE0CB6.xkm
[ 17262.941] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 17262.951] (II) XKB: reuse xkmfile /var/lib/xkb/server-81907AB388B478D711E6423DD8B0048239BE0CB6.xkm
[ 17262.964] (II) XKB: reuse xkmfile /var/lib/xkb/server-81907AB388B478D711E6423DD8B0048239BE0CB6.xkm
[ 17262.977] (II) XKB: reuse xkmfile /var/lib/xkb/server-81907AB388B478D711E6423DD8B0048239BE0CB6.xkm
[ 17262.993] (II) XKB: reuse xkmfile /var/lib/xkb/server-81907AB388B478D711E6423DD8B0048239BE0CB6.xkm
[ 17263.005] (II) XKB: reuse xkmfile /var/lib/xkb/server-81907AB388B478D711E6423DD8B0048239BE0CB6.xkm
[ 17263.022] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 17263.022] (II) XKB: reuse xkmfile /var/lib/xkb/server-81907AB388B478D711E6423DD8B0048239BE0CB6.xkm
[ 17263.073] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 17263.114] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 17263.155] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 17263.195] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 17263.237] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 17263.283] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 17263.326] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 17263.370] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 17263.409] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 17328.312] (II) XKB: reuse xkmfile /var/lib/xkb/server-F368D20DF3963C9041B5140771273B3C0AD33E46.xkm
[ 18630.621] (EE) 
[ 18630.621] (EE) Backtrace:
[ 18630.622] (EE) 0: /usr/bin/X (xorg_backtrace+0x48) [0x7fa74c0c0ed8]
[ 18630.622] (EE) 1: /usr/bin/X (0x7fa74bf1c000+0x1a8f19) [0x7fa74c0c4f19]
[ 18630.622] (EE) 2: /lib/x86_64-linux-gnu/libc.so.6 (0x7fa74a09c000+0x36d40) [0x7fa74a0d2d40]
[ 18630.622] (EE) 3: /usr/lib/xorg/modules/drivers/intel_drv.so (0x7fa745ea2000+0x463ca) [0x7fa745ee83ca]
[ 18630.622] (EE) 4: /usr/lib/xorg/modules/drivers/intel_drv.so (0x7fa745ea2000+0x4f475) [0x7fa745ef1475]
[ 18630.622] (EE) 5: /usr/bin/X (0x7fa74bf1c000+0x131288) [0x7fa74c04d288]
[ 18630.622] (EE) 6: /usr/bin/X (0x7fa74bf1c000+0x4ff0f) [0x7fa74bf6bf0f]
[ 18630.622] (EE) 7: /usr/bin/X (0x7fa74bf1c000+0x5369e) [0x7fa74bf6f69e]
[ 18630.622] (EE) 8: /usr/bin/X (0x7fa74bf1c000+0x574ea) [0x7fa74bf734ea]
[ 18630.622] (EE) 9: /lib/x86_64-linux-gnu/libc.so.6 (__libc_start_main+0xf5) [0x7fa74a0bdec5]
[ 18630.622] (EE) 10: /usr/bin/X (0x7fa74bf1c000+0x42a5e) [0x7fa74bf5ea5e]
[ 18630.622] (EE) 
[ 18630.622] (EE) Segmentation fault at address 0x7ffd04d58b00
[ 18630.623] (EE) 
Fatal server error:
[ 18630.623] (EE) Caught signal 11 (Segmentation fault). Server aborting
[ 18630.623] (EE) 
[ 18630.623] (EE) 
Please consult the The X.Org Foundation support 
	 at http://wiki.x.org
 for help. 
[ 18630.623] (EE) Please also check the log file at "/var/log/Xorg.0.log" for additional information.
[ 18630.623] (EE) 
[ 18630.623] (II) AIGLX: Suspending AIGLX clients for VT switch
[ 18631.838] (EE) Server terminated with error (1). Closing log file.
```
